x-pack/plugins/apm/public/components/app/Correlations/SignificantTermsTable.tsx

```diff
@@ -24,53 +26,47 @@ type SignificantTerm = NonNullable<
 interface Props<T> {
   significantTerms?: T[];
   status: FETCH_STATUS;
+  cardinalityColumnName: string;
   setSelectedSignificantTerm: (term: T | null) => void;
 }
 
 export function SignificantTermsTable<T extends SignificantTerm>({
   significantTerms,
   status,
+  cardinalityColumnName,
   setSelectedSignificantTerm,
 }: Props<T>) {
   const history = useHistory();
-  const columns = [
+  const columns: Array<EuiBasicTableColumn<T>> = [
     {
-      field: 'matches',
-      name: 'Matches',
+      width: '100px',
+      field: 'score',
+      name: 'Score',
       render: (_: any, term: T) => {
-        return (
-          <EuiToolTip
-            position="top"
-            content={`(${asInteger(term.fgCount)} of ${asInteger(
-              term.bgCount
-            )} requests)`}
-          >
-            <>
-              <EuiBadge
-                color={
-                  term.fgCount / term.bgCount > 0.03 ? 'primary' : 'secondary'
-                }
-              >
-                {asPercent(term.fgCount, term.bgCount)}
-              </EuiBadge>
-              ({Math.round(term.score)})
-            </>
-          </EuiToolTip>
-        );
+        return <EuiCode>{Math.round(term.score)}</EuiCode>;
+      },
+    },
+    {
+      field: 'cardinality',
+      name: cardinalityColumnName,
+      render: (_: any, term: T) => {
+        const matches = asPercent(term.fgCount, term.bgCount);
+        return `${asInteger(term.fgCount)} (${matches})`;
       },
     },
     {
       field: 'fieldName',
       name: 'Field name',
     },
     {
-      field: 'filedValue',
+      field: 'fieldValue',
       name: 'Field value',
       render: (_: any, term: T) => String(term.fieldValue).slice(0, 50),
     },
     {
-      field: 'filedValue',
-      name: '',
+      width: '100px',
+      field: 'filter',
```

Comment:
Minor nit but the implementation of the filter options bothers me. I think `EuiBasicTable` describes a specific way to get these type of icon buttons as actions in their [example](https://elastic.github.io/eui/#/tabular-content/tables#responsive-tables). Would it be possible to refactor this? That means you can get rid of the `/` between the icons.

Reply:
Good point. Done!